dominion/templates/profile/profile_detail.html

```diff
@@ -56,6 +24,22 @@
       <script src="{% static 'js/profile_map.js' %}"></script>
```

Comment:
let us remove `body_javascript` block  on profile html, as we have moved to mapit

Reply:
not sure what you mean